packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/calendar.tests.js

```diff
@@ -4617,6 +4618,52 @@ QUnit.module('Aria accessibility', {
         fx.off = false;
     }
 }, () => {
+    QUnit.test('aria-label should be \'Readonly-calendar\' and role = group when readOnly option is true', function(assert) {
+        const $element = this.$element;
+
+        $element.dxCalendar({
+            readOnly: true
+        }).dxCalendar('instance');
```

Comment:
Let's add an empty line to separate assert from initialazing?

Reply:
[1dca303](https://github.com/DevExpress/DevExtreme/pull/27898/commits/1dca303116defe32eec6021e83ea21a0d6819e16)